### PR TITLE
fix: monthly date offsets

### LINKED ISSUE #473: monthly next open date incorrect
When a resource is open on a monthly basis, e.g. "Open every 3rd Sunday from 7 - 8", the date displayed as the next-open date, is incorrect depending on the time of day.

## How to Recreate

This was noticed on the [St. Benedicts - Free Clinic at Josephine Commons](https://beta.upswyng.org/resource/609890192d902f0056fcf04d) resource.

1. navigate to the resource above
2. set your machine's date and time to May 10, 6:00pm
   - The date above is not significant, other than being used to display the bugged output below.
   - 6:00 pm is the earliest time I was able to use to recreate the issue. Anything sooner looks correct.
3. notice the next open date is a day before the actual date
   - For the date/time set in step 2, you should see "May 17 repeats every month on the 3rd Tuesday", but May 17 is the Monday before the correct date, May 18.

## Notes

- dates are stored separately from open/close times as [rrule](https://github.com/jakubroztocil/rrule) strings 
- the issue appears to be [where we create the next open date](https://github.com/CodeForBoulder/upswyng/blob/master/packages/web/src/components/MonthlySchedule.tsx#L43)

### PR DESCRIPTION
**Pull request by jacobvenable**
This PR closes #473 

## What does this PR do?

- fixes #473 
- removes some warnings in the dev console shown on the home page

## How does this PR make you feel? [:link:](http://giphy.com/categories/emotions/)

Jim Carrey looking at time, astonished

### CODE REVIEW

File: packages/web/src/components/MonthlySchedule.tsx [hunk 1]
```diff
@@ -40,14 +40,23 @@ const MonthlySchedule = ({ items }: { items: TScheduleItem[] }) => {
...
-          .all()[0]
-          .toLocaleString(undefined, { month: "short", day: "numeric" });
+        const nextOccurrenceDate = items[0].recurrenceRule.all()[0];
+        const nextOccurenceDateString = new Date(
+          nextOccurrenceDate
+        ).toLocaleString(undefined, {
+          month: "short",
+          day: "numeric",
+          // the rrule date is UTC-based
+          timeZone: "UTC",
```
Comment (jacobvenable) [inline packages/web/src/components/MonthlySchedule.tsx:50]: This was the main fix.

File: packages/web/src/components/HomeLink.tsx [hunk 2]
```diff
@@ -1,29 +1,20 @@
...
-  React.HTMLProps<HTMLAnchorElement>;
+type HomeLink = HomeLinkPropsBase & THomeButtonRouterLink;
 
-export const HomeRouterLink = (props: HomeRouterLinkProps) => {
-  const { children, linkProps, ...rest } = props;
+const HomeLink = (props: HomeLink) => {
+  const { children, linkProps } = props;
   return (
-    <Link {...linkProps} {...rest}>
+    <Link style={{ textDecoration: "none" }} {...linkProps}>
```
Comment (jacobvenable) [inline packages/web/src/components/HomeLink.tsx:14]: This update was just to get rid of some console warnings shown on the home page where improper props were being added to the DOM element.

Comment (jollyjerr) [reply packages/web/src/components/HomeLink.tsx:14]: I like it!

**Review by jollyjerr — APPROVED** [review]
:shipit: 🚢 🚀 🥇 :shipit:

**Comment by jollyjerr** [comment]
Uh oh @jacobvenable sorry if I stepped on your toes!

**Comment by jacobvenable** [comment]
> Uh oh @jacobvenable sorry if I stepped on your toes!

No worries it took me a bit to get around to fixing it 😆 I appreciate you doing that 🙂

**Pull request merged**